recipe/thirdparty/tpmodels/recipeInterface.go

```diff
@@ -18,13 +18,23 @@ package tpmodels
 import "github.com/supertokens/supertokens-golang/supertokens"
 
 type RecipeInterface struct {
-	GetUserByID             *func(userID string, userContext supertokens.UserContext) (*User, error)
-	GetUsersByEmail         *func(email string, userContext supertokens.UserContext) ([]User, error)
-	GetUserByThirdPartyInfo *func(thirdPartyID string, thirdPartyUserID string, userContext supertokens.UserContext) (*User, error)
-	SignInUp                *func(thirdPartyID string, thirdPartyUserID string, email string, userContext supertokens.UserContext) (SignInUpResponse, error)
+	GetUserByID                *func(userID string, userContext supertokens.UserContext) (*User, error)
+	GetUsersByEmail            *func(email string, userContext supertokens.UserContext) ([]User, error)
+	GetUserByThirdPartyInfo    *func(thirdPartyID string, thirdPartyUserID string, userContext supertokens.UserContext) (*User, error)
+	SignInUp                   *func(thirdPartyID string, thirdPartyUserID string, email string, oAuthTokens TypeOAuthTokens, rawUserInfoFromProvider TypeRawUserInfoFromProvider, userContext supertokens.UserContext) (SignInUpResponse, error)
+	ManuallyCreateOrUpdateUser *func(thirdPartyID string, thirdPartyIserID string, email string, userContext supertokens.UserContext) (ManuallyCreateOrUpdateUserResponse, error)
 }
```

Comment:
you need to add new recipe functions for multi tenant stuff.

Reply:
Done